addons/account/models/account.py

```diff
@@ -67,13 +132,14 @@ class AccountTaxReportLine(models.Model):
 
     name = fields.Char(string="Name", required=True, help="Complete name for this report line, to be used in report.")
     tag_ids = fields.Many2many(string="Tags", comodel_name='account.account.tag', relation='account_tax_report_line_tags_rel', help="Tax tags populating this line")
-    country_id = fields.Many2one(string="Country", comodel_name='res.country', required=True, default=lambda x: x.env.user.company_id.country_id.id, help="Country for which this line is available.")
     report_action_id = fields.Many2one(string="Report Action", comodel_name='ir.actions.act_window', help="The optional action to call when clicking on this line in accounting reports.")
-    children_line_ids = fields.One2many(string="Children Lines", comodel_name='account.tax.report.line', inverse_name='parent_id', help="Lines that should be rendered as children of this one")
+    children_line_ids = fields.One2many(string="Children Lines", comodel_name='account.tax.report.line', inverse_name='parent_id', copy=True, help="Lines that should be rendered as children of this one")
     parent_id = fields.Many2one(string="Parent Line", comodel_name='account.tax.report.line')
     sequence = fields.Integer(string='Sequence', required=True,
         help="Sequence determining the order of the lines in the report (smaller ones come first). This order is applied locally per section (so, children of the same line are always rendered one after the other).")
     parent_path = fields.Char(index=True)
+    parent_report_id = fields.Many2one(string="Tax Report", comodel_name='account.tax.report', ondelete='cascade', help="The parent tax report of this line, if it is directly at the root of the report")
+    report_id = fields.Many2one(string="Report", comodel_name='account.tax.report', compute='_compute_report_id', store=True, help="The tax report this line belongs to.")
```

Comment:
wouldn't be easier to say the report_id field is required? 

and parent_report_id is to be removed